updater.go

```diff
@@ -232,15 +234,18 @@ func (u *Updater) promptForUpdateAction(ctx Context, update Update, options Upda
 	auto, autoSet := u.config.GetUpdateAuto()
 	autoOverride := u.config.GetUpdateAutoOverride()
 	u.log.Debugf("Auto update: %s (set=%s autoOverride=%s)", strconv.FormatBool(auto), strconv.FormatBool(autoSet), strconv.FormatBool(autoOverride))
-	if auto && !autoOverride && !ctx.IsCheckCommand() {
-		isActive, err := u.checkUserActive(ctx)
-		if isActive {
-			err = fmt.Errorf("GUI is active, try later")
-		}
-		if err != nil {
-			return UpdateActionError, err
+	if auto && !autoOverride {
+		if !ctx.IsCheckCommand() {
+			isActive, err := u.checkUserActive(ctx)
+			if isActive {
+				err = fmt.Errorf("GUI is active, try later")
+				return UpdateActionUIBusy, err
+			}
+			if err != nil {
+				return UpdateActionError, err
+			}
```

Comment:
Woah do you think there's a reason this `if isActive` and `if err != nil` was in the wrong order before?

Reply:
Now that you've changed `checkUserActive` to actually return errors instead of just returning `err=nil` all the time. If that thing _fails_ the proper response is probably to go ahead and update. A faulty gui writing bogus app-state files shouldn't block upgrades forever.

What about something like this where we just ignore errors from `checkUserActive`?
```
isActive, err := u.checkUserActive(ctx)
if err == nil && isActive {
    return UpdateActionUIBusy, nil
}
u.guiBusyCount == 0
```

Reply:
ah of course, sorry. I added to the test for this case.